pages/controls/Certificate_and_Public_Key_Pinning.md

```diff
@@ -2,798 +2,127 @@
 
 title: Certificate and Public Key Pinning
 layout: col-sidebar
-author: Jeffery Walton, JohnSteven, Jim Manico, Kevin Wall, Ricardo Iramar
-contributors: Jack Mannino, Karl Fogel, Jshowalter , Achim, Pawel Krawczyk, Peter Bachman, Bill Sempf, Izar, Echsecutor, Jmanico, Douglasheld, Anant Shrivastava, Riramar, Nabla.c0d3, Neil Smithline, Tfrdidi, kingthorin
+author: Mark Gamache, Kevin Wall
+contributors: 
 tags: controls
 permalink: /controls/Certificate_and_Public_Key_Pinning
 
 ---
 
 {% include writers.html %}
 
-Certificate and Public Key Pinning is a technical
-guide to implementing certificate and public key pinning as discussed at
-the *[Virginia chapter's](https://owasp.org/www-chapter-northern-virginia/)*
-presentation [Securing Wireless Channels in the Mobile
-Space](https://wiki.owasp.org/images/8/8f/Securing-Wireless-Channels-in-the-Mobile-Space.ppt).
-This guide is focused on providing clear, simple, actionable guidance
-for securing the channel in a hostile environment where actors could be
-malicious and the conference of trust a liability.
+## Background
+Certificate and Public Key Pinning is a guide to understanding the current state of PKI security as it relates to certificate and public key pinning. Pinning was discussed at the [Virginia chapter’s](https://owasp.org/www-chapter-northern-virginia/) presentation [Securing Wireless Channels in the Mobile Space](https://wiki.owasp.org/images/8/8f/Securing-Wireless-Channels-in-the-Mobile-Space.ppt). The [previous version of this guide](https://web.archive.org/web/20230831033917/https:/owasp.org/www-community/controls/Certificate_and_Public_Key_Pinning) was focused on providing clear, simple, actionable guidance for securing the channel in a hostile environment as well as a look at the threat model that made pinning necessary. If one finds the need, that implantation information is still valid. However, this guide is focused on understanding the current state of public PKI and shows why, in nearly all cases, pinning creates much more risk than reward.
 
-A cheat sheet is available at [Pinning Cheat Sheet](https://cheatsheetseries.owasp.org/cheatsheets/Pinning_Cheat_Sheet.html).
+The document also is meant to clarify a point that was not made clear. If pinning is warranted, it should only be done when the client and server sides are both controlled by the same party. Trying to coordinate certificate rotations is a recipe for outages. Further, the original scope is intended for mobile applications only, which may have not been clear on older guidance.
```

Comment:
also is ➡️ is also
was not made clear. ➡️ was not made clear in the old OWASP article.
(Or something like that, currently the sentence is only a fragment)

Reply:
which may have not been clear ➡️ which may not have been clear